coalib/parsing/Globbing.py

```diff
@@ -29,6 +29,24 @@ def _end_of_set_index(string, start_index):
     return closing_index
 
 
+def glob_escape(string):
```

Comment:
I use `string` also as a variable name. Not sure this is so important, we don't use `string`-module at all in coala :) (at least I haven't seen it)